### ISSUE TITLE
TC 2 Server Install and custom world not working

### ISSUE THREAD

**Issue by MrSnuggies** [GreatOrator/TolkienCraftII#135]
Good evening sir.  I hate to bug you.  I have played your TC2 pack on single player before in the past.  I love it!  Great work btw!  I have a online server host now with Bisecthosting and your mod pack isn't a preloaded pack that I can just swap out on a dime.  I had them manually install your TC2 pack today.  When I log into the server, your awesome custom world is non existent.  It appears a randomly generated world is what I get.  Would you happen to have any additional steps to guide me to fixing this?

**Comment by teagan75** [comment]
I believe the issue is that on the server you need to rename the MiddleEarth folder to DIM100 in the world save

**Comment by GreatOrator** [comment]
either that or they did not install the world with the pack when they did it which means you will have to download and then upload to your server.

**Comment by MrSnuggies** [comment]
It appears that the host did not download the map and upload it as part of the install.  I actually downloaded it myself and attempted to upload it to the ftp myself but they cap me at 200meg.  So I had to open a trouble ticket and they are taking care of the world upload.  It does appear that DIM-100 is already named in the folder so I don't expect to have any issues with that.  I will give an update tomorrow when I get off work but it does appear that this will fix my issue.  I am not a big server guy when it comes to MC.  I am used to selecting pre selected mod packs from a drop down menu by the host.  Makes life a little easy if you know what I mean =)
